s3pd/__init__.py

```diff
@@ -103,6 +104,38 @@ def download_chunk(bucket, key, shmfileno, offset_first, offset_last, signed):
             Range='bytes=%s-%s' % (offset_first, offset_last))['Body']
         chunk._raw_stream.readinto(shmmap)
 
+def resolve_link(bucket, key, signed, depth=10):
+    # Stop after too many link indirections
+    assert depth > 0, 'Too many levels of link indirections'
+
+    client = create_client(signed)
+    filesize = get_filesize(client, bucket, key)
+
+    # There is no need to resolve files with a size >1KB, these could not
+    # realistically be links
+    if filesize > 1024:
+        return bucket, key
+
+    link_sentinel = '#S3PDLINK#'
```

Comment:
The link sentinel should be parametrized globally, so that the users can create the remote link file from the library.

Reply:
Agreed. As it's a custom behavior, the library should provide a mean to create/maintain those links too. (FAR)

Reply:
Fixed in 38d6c86 for exposing the sentinel.
API for handling symlink will be FAR.